be/src/vec/exec/scan/new_olap_scan_node.cpp

```diff
@@ -36,6 +36,7 @@ NewOlapScanNode::NewOlapScanNode(ObjectPool* pool, const TPlanNode& tnode,
 }
 
 Status NewOlapScanNode::prepare(RuntimeState* state) {
+    SCOPED_CONSUME_MEM_TRACKER(mem_tracker());
```

Comment:
`SCOPED_CONSUME_MEM_TRACKER(mem_tracker());` should be after `VScanNode::prepare`
_mem_tracker is initialized in `VScanNode::prepare -> ExecNode::prepare`,